server/assets/css/main.css

```diff
@@ -30,12 +30,12 @@ details summary > * {
   @apply pl-4;
 }
 
-.tree > li {
+.tree .node {
```

Comment:
Had to remove the child combinator to make the top-level skipped / not skipped attribute rules work, which then required switching to a class to not mess up other list items by accident.